src/main/java/org/opensearch/performanceanalyzer/rca/framework/metrics/WriterMetrics.java

```diff
@@ -145,7 +145,18 @@
                     Statistics.SUM)),
 
     STALE_METRICS("StaleMetrics", "count", Arrays.asList(Statistics.COUNT)),
-    ;
+
+    // This metrics indicates that the write file is not created.
+    // When there is no metrics to be written, the creation of writer file is skipped.
+    WRITER_FILE_NOT_CREATED(
```

Comment:
consider renaming this to `WRITER_FILE_CREATION_SKIPPED`

Reply:
Done